app/assets/stylesheets/common.scss

```diff
@@ -1,4 +1,5 @@
 @import "parameters";
+@import "bootstrap-custom";
```

Comment:
How does this find the other file when the file starts with an underscore? Is there some rule where the scss preprocessor searches for files with the underscore?

What's the logic behind preferring a CSS level `@import` over a sprockets level require? Then again we already seem to have a mix of both and I'm not sure how the split was decided...

Reply:
I'm just following the documentation, I don't have any particular views either way.